tkltest/execute/unit/execute.py

```diff
@@ -355,7 +355,10 @@ def merge_modules_coverage_reports(tkltest_config, modules_configs):
     modules_configs_with_exec_file = []
     # first collect exec files and app_paths
     for module_config in modules_configs:
```

Comment:
Can remove the check later on by changing the loop to iterate over `module_config in [m in modules_configs if m['general']['module_name'] not in failed_modules]`

Reply:
done